crates/bevy_sprite/src/mesh2d/mesh2d_types.wgsl

```diff
@@ -13,4 +13,5 @@ struct Mesh2d {
     local_from_world_transpose_b: f32,
     // 'flags' is a bit field indicating various options. u32 is 32 bits so we have up to 32 options.
     flags: u32,
+    tag: u32,
```

Comment:
Do we need any padding for web?

Reply:
I tested wasm but just realized it was webgpu, I'll check.

Reply:
Tested with webgl and seems fine.